web/hooks/use-users.ts

```diff
@@ -32,12 +31,9 @@ export const useRecentlyActiveUsersAndPrivateUsers = (limit: number) => {
   >()
   const loadUsers = async () => {
     const users = await getRecentlyActiveUsers(limit)
-    const privateUsers = await Promise.all(
-      users.map(async (u) => getPrivateUser(u.id))
-    )
```

Comment:
Why not load private users here with api here?

Reply:
i am not convinced the api will be authed when called in getServerSide props

Reply:
I mean useEffects are not run on the server so yes.

But this whole hook doesn't make sense. It can't be called something to get private users and then always return undefined lol. Then just remove the private users part.